spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -170,7 +169,8 @@ public SpannerEntityProcessor getSpannerEntityProcessor() {
 
 	public String applySortingPagingQueryOptions(SpannerQueryOptions options,
 			String sql) {
-		StringBuilder sb = applySort(options.getSort(), wrapAsSubSelect(sql));
+		StringBuilder sb = SpannerStatementQueryExecutor.applySort(options.getSort(),
+				wrapAsSubSelect(sql), o -> o.getProperty());
 			if (options.hasLimit()) {
```

Comment:
Wrong indentation.

Reply:
I don't think so, I tried moving this to the previous line and the auto format moves it back to being like this.